lib/bitex/payment.rb

```diff
@@ -12,37 +15,47 @@ def self.from_json(json)
         json.each do |key,raw_value|
           next if raw_value.nil?
 
-          value = if [:valid_until, :quote_valid_until, :last_quoted_on].include?(key.to_sym)
+          value = if %i[valid_until quote_valid_until last_quoted_on].include?(key.to_sym)
             Time.at(raw_value)
           else
             raw_value
           end
-          thing.send("#{key}=", value) rescue nil
+          begin
+            thing.send("#{key}=", value)
+          rescue StandarError
```

Comment:
Tests should not have passed with this typo in place, that means this piece of code is not being tested. We should look into this.